velox/functions/lib/aggregates/ValueList.cpp

```diff
@@ -65,7 +65,7 @@ void ValueList::appendNonNull(
   allocator->extendWrite(dataCurrent_, stream);
   // The stream may have a tail of a previous write.
   const auto initialSize = stream.size();
-  static const exec::ContainerRowSerdeOptions options{};
```

Comment:
Perhaps, add comments to clarify the meaning of 'false' and the reason for choosing that value. While you are changing this file, would you add a new line before this variable and rename options to kOptions to match coding style?

```

// Set isKey to false to avoid unnecessary sorting.
static const exec::ContainerRowSerdeOptions kOptions{/*isKey=*/false};
```